src/components/import/badges/Badge.scss

```diff
@@ -0,0 +1,17 @@
+.ocs-badge {
```

Comment:
Think we could get rid of the ocs prefix and use our own prefix

Reply:
Sure, prefix issue will be taken care of in [GITOPS-2291](https://issues.redhat.com/browse/GITOPS-2291). I will mark this down. Thanks for the catch 👍